controller.go

```diff
@@ -397,3 +402,14 @@ func (c *controller) isDriverGlobalScoped(networkType string) (bool, error) {
 func (c *controller) GC() {
 	sandbox.GC()
 }
+
+func (c *controller) scope(networkType string) (driverapi.Scope, error) {
```

Comment:
I dont think it is a good idea to have a scope function per networkType.
I think this should be per object. To be precise, I think we should add a `StoreScope` function in `datastore.KV` interface. Individual objects should decide if it is local or global. 
For example, we will be including sandbox as an object  in the datastore. The sandbox objects will be local even for those networks that are global in nature.